support-models/src/main/scala/com/gu/support/workers/states/SendThankYouEmailState.scala

```diff
@@ -2,15 +2,16 @@ package com.gu.support.workers.states
 
 import java.util.UUID
 
-import com.gu.support.workers.{PaymentMethod, SalesforceContactRecord, User, _}
+import com.gu.support.workers.{SalesforceContactRecord, User, _}
 import com.gu.support.encoding.CustomCodecs.{decodeLocalTime, encodeLocalTime}
 import org.joda.time.LocalDate
 
+
 case class SendThankYouEmailState(
   requestId: UUID,
   user: User,
   product: ProductType,
-  paymentMethod: PaymentMethod,
+  paymentMethod: PaymentMethodDisplayFields,
```

Comment:
before merging I'll rename the paymentMethod field here and in the acquisitionsEventState to displayPaymentMethod to make it different from the full payment method object that is in createSubscriptionState.
Unless anyone has of any reason not to do that